train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -45,5 +48,18 @@ private void enforceSpeedLimit() {
 	public void setJoystickPosition(int joystickPosition) {
 		this.step = joystickPosition;		
 	}
+	
+	class UpdateTask extends TimerTask {
+        public void run() {
+            followSpeed();
+        }
+	}
+	
+	public TrainControllerImpl(){
+		
+		Timer speedUpdater = new Timer();
+		speedUpdater.scheduleAtFixedRate(new UpdateTask(), 0, 100);
```

Comment:
Is not hard coding would be a better idea or the timing can be seen as a constatn value?